tests/src/test/java/edu/snu/onyx/tests/examples/ExampleTestUtil.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2017 Seoul National University
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.snu.onyx.tests.examples;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public final class ExampleTestUtil {
+  public static boolean isOutputSame(String resourcePath, String outputFileName, String testResourceFileName) throws IOException {
```

Comment:
I think `Valid` or `Correct` would be more suitable than `Same`, vocabulary-wise.

Reply:
@wonook Fixed. Thank you!